### ISSUE TITLE
URP Help

### ISSUE THREAD

**Issue by dan-hansen** [Ooseykins/QuickDither#2]
Hi, this is just what I've been looking for. I got this working with default rendering, but the instructions for URP are not detailed enough for my limited experience with it.

Thus far, I installed URP and followed directions to convert materials, etc.  Thus, the basic URP s setup is working elsewhere in my project.  When I load up your example scene, it doesn't appear to be working yet and has a few warnings about missing scripts referenced. P.s. I am just throwing all the assets into the Asset folder.  I attempted to follow the setup instructions, but I just get a white screen. It's not really clear to me how to satisfy the missing scripts or where to go for most of the tips provided.

Beyond this, I need to get it working with 2D rendering/lights with a two-camera setup that already exists as part of another Unity package I'm using. I'm betting the author of said package can help sort it out, but I'd like to clarify here first. 

Thanks!

**Comment by Ooseykins** [comment]
You might have better luck using the release package (on the right side from the main page of this github) and import that into Unity. Either delete or just don't import the Standard folder inside of Pipelines, that contains code that will reference missing scripts when used in URP.

As for the URP specific usage, the example scene can help to set it up. You must be rendering to a render texture [(docs here)](https://docs.unity3d.com/Manual/class-RenderTexture.html) which you can do with the Output settings on your Camera component. To get your game appearing again, you apply that render texture to a Raw Image component on a canvas (example in the scene), then apply the dither material to that Raw Image component to get the effect.

**Comment by dan-hansen** [comment]
Thanks, I am able to get it working partially now. However, now I have to figure out how to not mess up the existing camera setup.  I already have a camera with a canvas, so when I drop in the raw image/other setup various elements shrink and/or do not display. I can probably get other support from here to figure out what is being impacted.

Let me know if you have other ideas/suggestions.  My suspicion is the 320x180 resolution (which is the final) on my dither render texture and order of rendering with layers.

![image](https://user-images.githubusercontent.com/77889523/213880713-e2e1072c-88e9-4907-aa4e-58fe583cdde0.png) - e.g. display text not getting rendered correctly.

**Comment by Ooseykins** [comment]
I think you're able to just use multiple canvases, if that helps your problem. If your existing canvas is camera-space the new (dithered) one should just be set a farther distance away.

**Comment by Ooseykins** [comment]
Also if you're building off the example: the render texture in the example should have it's filter mode set to point and it is not. I can't tell from your screenshot though, so looks like you got it right.

**Comment by dan-hansen** [comment]
FYI, I was never able to work out my multiple camera issue (might look into eventually) and for now am relying on something I figured out with using URP 2D Render features documented here -  https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/master/Assets/Scripts/Editor . This essentially allowed me to apply your dither material to the final rendering output of my selected camera sorting layer. This seemed to work pretty well and I didn't have to solve whatever is going on my intermediate camera setup yet. If you want to explore further could connect... otherwise I was hesitant to ask the package developer for help on your documented setup for URP quite yet. Above is interesting and might be an alternate solution provided URP continues to support it one fashion or another.  The example code for the custom render feature I used isn't maintained any longer.
